beacon-chain/sync/initial-sync/initial_sync_test.go

```diff
@@ -292,9 +290,7 @@ func connectPeerHavingBlocks(
 	})
 
 	p.SetStreamHandler("/eth2/beacon_chain/req/beacon_blocks_by_root/1/ssz_snappy", func(stream network.Stream) {
-		defer func() {
-			assert.NoError(t, stream.Close())
-		}()
+		defer stream.Close()
```

Comment:
Please handle this error from stream.Close(). (Static analysis will not allow the build to complete with unhandled errors)